docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,125 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* Add `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24604`)
```

Comment:
Do we want this in the release notes at all.. I think not @martint

Reply:
This is something exposed for users of JDBC driver

Reply:
ok .. will leave then.